packages/web-components/src/dialog/dialog.styles.ts

```diff
@@ -6,116 +6,98 @@ import {
   colorNeutralBackground1,
   colorNeutralForeground1,
   colorTransparentStroke,
-  fontFamilyBase,
-  fontSizeBase300,
-  fontSizeBase500,
-  fontWeightRegular,
-  fontWeightSemibold,
-  lineHeightBase300,
-  lineHeightBase500,
+  curveAccelerateMid,
+  curveDecelerateMid,
+  curveLinear,
+  durationGentle,
   shadow64,
-  spacingHorizontalS,
-  spacingHorizontalXXL,
-  spacingVerticalS,
-  spacingVerticalXXL,
   strokeWidthThin,
 } from '../theme/design-tokens.js';
 
 /** Dialog styles
  * @public
  */
 export const styles = css`
-  ${display('flex')}
+  @layer base {
```

Comment:
I *think* we have broad enough support here given the baseline. Would be great to identify when/where we leverage cascade layers and get that documented at some point for contributors!